src/cogent3/app/io.py

```diff
@@ -362,6 +378,10 @@ def write(self, data, identifier=None):
 class write_seqs(_checkpointable):
     """Writes sequences to text files in standard format."""
 
+    _input_types = ("sequences", "aligned")
```

Comment:
You must use the TYPES defined in composable! Quite clearly, the original code was also wrong, i.e. `input_types=SEQUENCE_TYPE`.